sql/step_a110_update_operations_table_change_ope_for_to_enum.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE OH_OPERATION MODIFY COLUMN OPE_FOR VARCHAR(20);
+
+UPDATE OH_OPERATION
+    SET OPE_FOR='opd_admission' WHERE OPE_FOR='1';
+UPDATE OH_OPERATION
+    SET OPE_FOR='admission' WHERE OPE_FOR='2';
+UPDATE OH_OPERATION
+    SET OPE_FOR='opd' WHERE OPE_FOR='3';
+
+ALTER TABLE OH_OPERATION
+    MODIFY COLUMN OPE_FOR ENUM('opd', 'admission', 'opd_admission') DEFAULT 'opd_admission';
```

Comment:
the order matters, please use this one:
```suggestion
    MODIFY COLUMN OPE_FOR ENUM('opd_admission', 'admission', 'opd') DEFAULT 'opd_admission';
```